### ISSUE TITLE
Title :- Pharmaceutical Drugs and Vitamins Images Detection

### ISSUE THREAD

**Issue by aaradhyasinghgaur** [Recode-Hive/machine-learning-repos#221]
**Description :-** The below dataset includes images of popular pharmaceutical drugs and vitamins in the Philippines. This is dataset can be used for classifying drug images using CNN and transfer learning. Currently, there are ten available classes of pill images.
                           One of the primary uses would be to assist individuals in identifying medications and vitamins. This could be helpful for patients who have multiple prescriptions or for caregivers who need to manage medications for others. By taking a picture of the medication, the system could provide information about its name, dosage, and usage instructions.
                           
**Datset I'll use :-** https://www.kaggle.com/datasets/vencerlanz09/pharmaceutical-drugs-and-vitamins-dataset-v2

**Solution I propose :-**
   1. Utilizing 5 models such as DenseNet121 , Xception, VGG16, ResNet50, and InceptionV3 for image classification.
   2. Applying data augmentation (rotation, zooming, flipping, shearing, brightness) to enhance dataset robustness.
   3. Comparing model performance using accuracy scores, loss/accuracy graphs, and confusion matrices.
   4. Conducting EDA for dataset insights, including image distribution, quality, class imbalances, and sample visualization.
   5. Documening the process in a comprehensive README file.
   
**Tools I'll use :-** Keras , numpy , matplotlib , scikit-learn , tqdm etc.

**Kindly assign this issue to me @sanjay-kv  and lable it as level-3 if possible as I'm training and testing dataset for 5 different models which is resource and time consuming and also making doucumentation about which works better . I will also be doing EDA analysis for the dataset.**
